portfolio/src/main/webapp/script.js

```diff
@@ -40,15 +40,20 @@ function addRandomStock() {
 }
 
 // Fetches comments from the server and adds them to the DOM.
-function LoadComments() {
-  const max = getMax();
-  fetch('/data?numCommentsToLoad=' + max).then(response => response.json()).then((comments) => {
-  const commentListElement = document.getElementById('comment-list');
-  commentListElement.innerHTML = ' ';
-  comments.forEach((comment) => {
-  commentListElement.appendChild(createListElement(comment));
+async function LoadComments() {
```

Comment:
```suggestion
async function loadComments() {
```

We typically use lower camel case for method names: https://google.github.io/styleguide/jsguide.html#naming-method-names

Reply:
addressed